sql/database.sql

```diff
@@ -107,13 +107,16 @@ CREATE TABLE `api_log` (
 
 DROP TABLE IF EXISTS `api_token`;
 CREATE TABLE `api_token` (
```

Comment:
getting following error on install:
```
PHP Fatal error:  Uncaught Exception: ERROR: unable to execute SQL: 'CREATE TABLE `api_token` (  `id` bigint(20) NOT NULL,  `user_id` varchar(40) DEFAULT NULL,  `token` varchar(128) DEFAULT NULL,  `expiry` datetime DEFAULT NULL,  `token_auth` text,  `token_api` varchar(4) DEFAULT NULL,  `patient_id` bigint(20) NOT NULL,  `client_id` varchar(80) DEFAULT NULL,  `auth_user_id` varchar(80) DEFAULT NULL,  `scope` text COMMENT 'json encoded'  PRIMARY KEY (`id`),  UNIQUE KEY `token` (`token`)) ENGINE = InnoDB' due to: You have an error in your SQL syntax; check the manual that corresponds to your MariaDB server version for the right syntax to use near '(`id`),  UNIQUE KEY `token` (`token`)) ENGINE = InnoDB' at line 1
```
Guessing it has to do with removing the `AUTO_INCREMENT` in the `id` .